cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/PipelineTable/PipelineTable.scss

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import '~styles/colors.scss';
+
+$table-bg-color: white;
+$table-border-color: $grey-06;
+$table-border: 1px solid $table-border-color;
+$row-hover-color: $grey-08;
+
+$type-width: 100px;
+$runs-width: 100px;
+$status-width: 150px;
+$last-start-width: 200px;
+$next-run-width: 150px;
+$action-width: 60px;
+$col-width: calc((100% - #{$type-width} - #{$runs-width} - #{$status-width} - #{$last-start-width} - #{$next-run-width} - #{$action-width}) / 2);
+
+.pipeline-list-table {
+  background-color: $table-bg-color;
+  padding: 0 4px;
+
+  .table-header {
+    height: 40px;
+    font-weight: bold;
+    border-bottom: 3px solid $table-border-color;
+    display: flex;
+    align-items: center;
+    line-height: 1;
+  }
+
+  .table-column {
+    padding: 0 5px;
+
+    &:first-child {
+      padding-left: 25px;
+    }
+
+    &:last-child {
+      padding-right: 25px;
+    }
+  }
+
+  .table-row-container {
+    border-bottom: $table-border;
+
+    &.expanded {
+      background-color: $row-hover-color;
+
+      .table-row:hover .pipeline-run,
+      .pipeline-run {
+        color: $orange-01;
+      }
+    }
+  }
+
+  .table-row {
+    min-height: 55px;
+    display: flex;
+    align-items: center;
+    width: 100%;
+
+    &:hover {
+      background-color: $row-hover-color;
+
+      .pipeline-run { color: $blue-02; }
+    }
+  }
+
+  .name {
+    width: $col-width;
+
+    .pipeline-name {
+      font-size: 16px;
```

Comment:
Same here as well. Could we use a heading instead of custom font size. @leacd.

Reply:
This could just be me but seeing different font sizes in the same table looks a little off to me. Do we really need to increase the font size of the pipeline name in the table @leacd ?

![screen shot 2018-12-03 at 11 11 44 am](https://user-images.githubusercontent.com/1452845/49395802-55fa9380-f6ec-11e8-9e6b-f6dd87a7fa1a.png)

Reply:
Talked with @leacd, we are going to keep the pipeline name bigger